notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6320 @@
+{
```

Comment:
<p>"Load Hugging face model..."</p><p> </p><p>"This step loads and deploys the Hugging Face..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>👍</p>